general/g.version/main.c

```diff
@@ -140,26 +144,18 @@ int main(int argc, char *argv[])
     }
 
     if (gish_rev->answer) {
-	char **rev_ver = G_tokenize(GIS_H_VERSION, "$");
-	char **rev_time = G_tokenize(GIS_H_DATE, "$");
-	const int tokens_expected = 3;
+	char *rev_ver = GIS_H_VERSION;
+	char *rev_time = GIS_H_DATE;
 	int no_libgis = FALSE;
 
-	/* if number of tokes is right, print it */
-	if (G_number_of_tokens(rev_ver) == tokens_expected &&
-	    G_number_of_tokens(rev_time) == tokens_expected) {
+	if (*rev_ver && *rev_time) {
 	    if (shell->answer) {
-                const char *p;
-                p = strstr(rev_ver[1], " ");
-		fprintf(stdout, "libgis_revision=%s\n",
-			p ? p + 1 : "00000");
-                p = strstr(rev_time[1], " ");
-		fprintf(stdout, "libgis_date=\"%s\"\n",
-			p ? p + 1 : "?");
+		fprintf(stdout, "libgis_revision=\"%s\"\n", rev_ver);
```

Comment:
@metzm Let's avoid quotes here.

Reply:
Are the quotes doing any harm?

Reply:
No, but it's just inconsistent...

```
g.version -gre
...
revision=35f284289
...
libgis_revision="5872054b4"
```

Reply:
@metzm It seems to be the last issue with this PR. Can we solve it in order to include it in release 7.8.3 ? Thanks!

Reply:
I agree. The quotes are now not needed and should be removed for consistency.